bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -1,346 +0,0 @@
-
-{% from "macros-protocol.html" import feature_card with context %}
-{% from "macros.html" import fxa_email_form, fxa_link_fragment, fxa_cta_link, google_play_button with context %}
-
-{% extends "firefox/base/base-protocol.html" %}
-
-{% set _entrypoint = 'mozilla.org-firefox-accounts' %}
-
-{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
-{% block page_desc %}
-{% if l10n_has_tag('firefox_send_20190420') %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}
-{% else %}
-  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.') }}
-{% endif %}
-{% endblock %}
-
-{% block page_css %}
- {{ css_bundle('firefox_accounts') }}
-{% endblock %}
-
-{% block site_header %}
-  {% with hide_nav_get_account_button=True %}
-    {% include 'includes/protocol/navigation/menu-mozilla/index.html' %}
-  {% endwith %}
-{% endblock %}
-
-{% block content %}
-
-<section class="mzp-c-hero mzp-has-image mzp-t-firefox mzp-t-product-firefox">
-  <div class="mzp-l-content">
-    <div class="mzp-c-hero-body">
-      {% if l10n_has_tag('firefox_send_20190420') %}
-        <h1 class="mzp-c-hero-title">{{ _('One login. Power and privacy everywhere.') }}</h1>
-      {% else %}
-        <h1 class="mzp-c-hero-title">{{ _('One log-in. Power and privacy everywhere.') }}</h1>
-      {% endif %}
-      <div id="hero-cta" class="mzp-c-hero-cta">
-        <div class="show-fxa-supported-signed-out">
-          {{ fxa_cta_link(entrypoint=_entrypoint, account_id='features-hero-account', action='signup', button_text=_('Create a Firefox Account'), button_class='mzp-c-button mzp-t-primary mzp-t-product', utm_campaign='fxa-benefits-page', utm_content='accounts-page-top-cta') }}
-        </div>
-        <div class="show-fxa-supported-signed-in">
-          <button class="mzp-c-button mzp-t-product" type="button" id="fx-modal" data-button-name="download the firefox app" data-cta-position="primary cta">{{ _('Download the Firefox App') }}</button>
-        </div>
-        <div class="show-fxa-ios">
-          <a class="mzp-c-button mzp-t-secondary" rel="external" href="https://support.mozilla.org/kb/sync-firefox-bookmarks-and-browsing-history-iOS">{{ _('Set up Sync for iOS') }}</a>
-        </div>
-        {# L10n: Line break below for visual formatting only. #}
-        <div class="show-fxa-android">
-          <a class="mzp-c-button mzp-t-secondary" rel="external" href="https://support.mozilla.org/kb/sync-bookmarks-tabs-history-and-passwords-android">{{ _('Set up Sync on your <br>Android device') }}</a>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="mzp-c-hero-image">
-    {{ high_res_img('firefox/accounts/fxa-hero-img.png', {'alt': '', 'width': '1200', 'height': '770'}) }}
```

Comment:
I believe this (and some of the other images referenced in this template) are no longer needed and can be removed. Please check carefully through the images before removing them.

https://github.com/mozilla/bedrock/tree/master/media/img/firefox/accounts